UnitTests/TextValidateFieldTests.cs

```diff
@@ -0,0 +1,561 @@
+﻿using System.Diagnostics;
```

Comment:
Nice job!!

NetMaskedTextProvider - 85.4% Code Coverage
TextMaskProvider - 57.3%
TextRegExProvider - 81.9%
TextValidateField - 85.4%

Really impressive work! Thank you.